### ISSUE TITLE
Release a Python package on https://pypi.org for much easier installation

### ISSUE THREAD

**Issue by ndbroadbent** [ndbroadbent/icloud_photos_downloader#85]
Do this after #82 (rewrite to clean up the code)

- [ ] Release our fork of `pyicloud` as a new package: `pyicloud-ipd`. To avoid all the version conflicts. (Would be better if we can get stuff merged into `pyicloud`.)
- [ ] Build a python package for `icloud_photos_downloader`, which installs an executable named: `icloudpd`.

Then people can easily install the script by running: `sudo pip install icloudpd`.

**Comment by ndbroadbent** [comment]
Done! You can now run: `sudo pip install icloudpd`
